client/my-sites/checkout/checkout-thank-you/pending/index.tsx

```diff
@@ -72,6 +73,7 @@ function CheckoutPending( {
 	receiptId,
 	siteSlug,
 	redirectTo,
+	fromSite,
```

Comment:
would it make sense to use the same parameter name for `fromSite` and `fromSiteSlug`? (If they are the same thing that is 🤔 )